index.html

```diff
@@ -185,5 +185,25 @@ <h2 class="third-text"><a>let's connect</a></h2>
           </li>
         </ul>
     </section>
+    <section class="contact-form">
+        <ul>
+            <li><img class="ellipse-12-2" src="elements/ellipse-12-2.png" alt="icon-7"></li>
+            <li><img class="ellipse-5-w" src="elements/ellipse-5-white.png" alt="icon-8"></li>
+            <li><img class="ellipse-3-w" src="elements/ellipse-3-white.png" alt="icon-9"></li>
+            <li><img class="rect-4-d" src="elements/rectangle-4-dark.png" alt="icon-10"></li>
+            <li><img class="rect-7-2" src="elements/rectangle-7-2.png" alt="icon-11"></li>
+        </ul>
+        <h2 class="contact-title">Contact me</h2>
+       <h3 class="contact-subtitle">If you have an application you are interested in developing, 
+           a feature that you need built or a project that needs coding.
+            I’d love to help with it 
+        </h3>
+        <form action="https://formspree.io/f/mdoyznvk" method="post">
+            <input type="text" class="input" required maxlength="30" placeholder="Yeremias">
+            <input type="email" class="input" required maxlength="30" placeholder="natajayanj14@gmail.com">
+            <textarea class="textarea" required maxlength="500" placeholder="Write your message here"></textarea>
+            <input type="submit" class="contact-button" value="Get in touch">
```

Comment:
- [ ] **OPTIONAL**
Kindly keep in mind that the `name` attribute in the `input` tags is very important by default. Because when the form is submitted, it carries the data that's been submitted by a user in each input field to somewhere, then defines it with the `name` attribute.
Example:
```
<input name="full name" type="text" class="input" required maxlength="30" placeholder="Yeremias">
value = "Mahmoud Bakr"
// outcome data
full name = "Mahmoud Bakr"
```
And the same way goes with the email which you should have used the formspree default which is `name="_replyto"`
Finally, for the `textarea`, should take a `name="message"` 

![Screenshot](https://user-images.githubusercontent.com/47625407/121280466-9d25f880-c8d6-11eb-8509-0f382e42e3a7.png)

> Please notice that this is a hard requirement for the form to work properly, and I'm sorry that it didn't show up for you in the earlier review.